s3store/s3store.go

```diff
@@ -60,14 +60,16 @@
 //
 // Therefore, a few special conditions have been implemented:
 //
-// Each PATCH request must contain a body of, at least, 5MB. If the size
-// is smaller than this limit, the entire request will be dropped and not
-// even passed to the storage server. If your server supports a different
-// limit, you can adjust this value using S3Store.MinPartSize.
+// Each PATCH request must contain a body of, at least, 5MB. If your server
```

Comment:
> Each PATCH request must contain a body of, at least, 5MB.

This is not correct anymore, is it? A PATCH request now can have any size.

Reply:
Ah, yes. Good point.